Src/AutoFixture.xUnit2/MemberAutoDataAttribute.cs

```diff
@@ -67,5 +74,25 @@ protected MemberAutoDataAttribute(DataAttribute autoDataAttribute, string member
         /// Gets the parameters passed to the member. Only supported for static methods.
         /// </summary>
         public IEnumerable<object> Parameters { get; }
+
+        /// <summary>
+        /// Returns the composition of test data from the member on the test class
+        /// combined with the auto-generated data specimens generated by AutoFixture.
+        /// </summary>
+        /// <param name="testMethod">The method that is being tested.</param>
+        /// <returns>
+        /// Returns the composition of the theory data provided by MemberDataAttribute and AutoFixture.
+        /// </returns>
+        public override IEnumerable<object[]> GetData(MethodInfo testMethod)
+        {
+            if (testMethod == null) throw new ArgumentNullException(nameof(testMethod));
+
+            var memberData = this.MemberDataAttribute.GetData(testMethod).ToList();
+
+            return from memberValues in memberData
+                   from autoValues in this.DataAttributeFactory.Invoke().GetData(testMethod).Take(1)
```

Comment:
It's a minor thing, but I feel that extracting this into a local variable called  `autoData` just below [L90](https://github.com/AutoFixture/AutoFixture/pull/1164/files#diff-8e03ae9ef1d63ccaa63555f169242392R90) would help clarify the relation between `memberData` and `autoData`.